TTS/bin/train_gan_vocoder.py

```diff
@@ -443,7 +441,8 @@ def main(args):  # pylint: disable=redefined-outer-name
     print(f" > Loading wavs from: {c.data_path}")
     if c.feature_path is not None:
         print(f" > Loading features from: {c.feature_path}")
-        eval_data, train_data = load_wav_feat_data(c.data_path, c.feature_path, c.eval_split_size)
+        eval_data, train_data = load_wav_feat_data(
```

Comment:
do you have any updates in this file rather than formatting?

Reply:
No changes made. Thought i reverted the formatting, sorry.